dotCMS/src/main/java/com/dotcms/graphql/business/GraphqlAPIImpl.java

```diff
@@ -179,7 +187,11 @@ private void createSchemaType(ContentType contentType,
 
             if(!(field instanceof RowField) && !(field instanceof ColumnField)) {
                 if (field instanceof RelationshipField) {
-                    handleRelationshipField(contentType, builder, field, graphqlObjectTypes);
+                    try {
+                        handleRelationshipField(contentType, builder, field, graphqlObjectTypes);
+                    } catch(Exception e) {
```

Comment:
do we really need catch Exception?, isn't it possible catch another more specific exception?